WcaOnRails/spec/controllers/api_competitions_controller_spec.rb

```diff
@@ -267,6 +275,54 @@
         expect(parsed_body).to eq(
           "formatVersion" => "1.0",
           "id" => "TestComp2014",
+          "events" => [
+            {
+              "id" => "333",
+              "rounds" => [
+                {
+                  "id" => "333-1",
+                  "format" => "a",
+                  "timeLimit" => {
+                    "centiseconds" => TimeLimit::TEN_MINUTES_IN_CENTISECONDS,
+                    "cumulative" => false,
```

Comment:
I think this should be an empty array of round ids :) (same goes for the few other test cases below)